IAMOTZ/ievents
# #152797059 Set up sequelize

#### What does this PR do?
It set up Sequelize ORM to interface with our Postgres database.
#### Description of Task to be completed?
Set up Sequelize
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Create a `.env` file to produce all the environment variables used in `./server/config/config.js`
- run `npm run server:dev`
- the server should start on port 3000
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152797059
#### Screenshots (if appropriate)
None
#### Questions:
None
